test/inductor/test_pattern_matcher.py

```diff
@@ -635,6 +635,63 @@ def forward(self, x):
 
         self.assertEqual(res1, res2)
 
+    def test_addmm_activation(self):
+        def fn_addmm_relu(input, mat1, mat2):
+            return torch.nn.functional.relu(torch.addmm(input, mat1, mat2))
+
+        def fn_addmm_gelu(input, mat1, mat2):
+            return torch.nn.functional.gelu(
+                torch.addmm(input, mat1, mat2), approximate="tanh"
+            )
+
+        def fn_add_mm_relu(input, mat1, mat2):
+            return torch.nn.functional.relu(torch.add(input, torch.mm(mat1, mat2)))
+
+        def fn_add_mm_gelu(input, mat1, mat2):
+            return torch.nn.functional.gelu(
+                torch.add(input, torch.mm(mat1, mat2)), approximate="tanh"
+            )
+
+        args = [
+            torch.randn(20, device="cuda"),
```

Comment:
Hi， this test suit is shared by `cuda` and other GPUs like `xpu`, please replace the hard code `cuda` with `GPU_TYPE` so that it will not fail on other GPUs.